content/installation/rust.md

```diff
@@ -6,4 +6,147 @@ implementation: unofficial
 weight: 1200
 ---
 
-Please see the [Cucumber-Rust website](https://github.com/bbqsrc/cucumber-rust).
+## Cucmber-Rust
+
+Cucumber-Rust is an unofficial implementation of the Cucumber testing framework for Rust. Fully native, no external test runners or dependencies.
+
+### Usage
+
+Create a directory called `tests/` in your project root and create a test target of your choice. In this example we will name it `cucumber.rs`.
+
+Add this to your `Cargo.toml`:
```

Comment:
Has this file been auto-generated somehow above that I've missed....

For example in ruby you can do `cucumber --init`

Reply:
In this case, you should do it by hands. I changed description a little to make it more clear.